src/client/scenes/Admin/scenes/Topics.js

```diff
@@ -1,10 +1,26 @@
 import React from "react";
 import { Grid, Row, Col } from "react-bootstrap";
 import APIProvider from "utils/APIProvider";
+import { connect } from "react-redux";
 
 import Alert from "react-s-alert";
 import Loader from "components/Loader/Loader";
 
+import {
+  addTopics,
+  startLoadingTopics,
+  stopLoadingTopics,
+  setCurrentTopic,
+  emptyCurrentTopic
+} from "state/actions/topic";
+import store from "state/store";
+import {
+  loadTopicsPage,
+  disposeTopicsPage,
+  loadEditTopic,
+  disposeEditTopic
+} from "state/actions/sagaActions";
+
 import ItemList from "../components/ItemList";
```

Comment:
It would probably make sense to have ItemListContainer and ItemFormContainer components, but since we're organizing this later, we could deal with this later.

Reply:
👍